### ISSUE TITLE
"Click below to continue to Jellyfin." does nothing

### ISSUE THREAD

**Issue by rigrig** [hrfee/jfa-go#26]
**Describe the bug**

After creating an account through an invite link, clicking the "Continue" button does nothing.

**To Reproduce**

1) create a new invite link
2) use the invite link to create a new account
  -> Success! (this works fine, the account was created)
3) press the "continue" button
Expected result: redirect to Jellyfin
Actual result: nothing happens

![image](https://user-images.githubusercontent.com/2941436/104074299-08787b00-5210-11eb-8389-4c3b07dc5ba4.png)

**Logs**
Debug logs:
```
[GIN/DEBUG] 00:16:27: GET(/invite/aj6DvYcTrB8z3uPtw3xXZ7) => 200 in 2.719397ms; 
[GIN/DEBUG] 00:16:28: GET(/invite/js/modules/modal.js) => 200 in 209.141µs; 
[GIN/DEBUG] 00:16:28: GET(/invite/js/modules/common.js) => 200 in 340.531µs; 
[GIN/DEBUG] 00:16:28: GET(/invite/css/remixicon.css) => 200 in 568.81µs; 
[GIN/DEBUG] 00:16:28: GET(/invite/css/modal.css) => 200 in 231.104µs; 
[GIN/DEBUG] 00:16:28: GET(/favicon-32x32.png) => 200 in 193.891µs; 
[GIN/DEBUG] 00:16:28: GET(/apple-touch-icon.png) => 200 in 202.703µs; 
[DEBUG] 00:16:28 daemon.go:37: Daemon: Checking invites
[DEBUG] 00:16:40 auth.go:89: Auth succeeded
[DEBUG] 00:16:40 api.go:871: Users requested
[DEBUG] 00:16:40 auth.go:89: Auth succeeded
[DEBUG] 00:16:40 api.go:686: Invites requested
[DEBUG] 00:16:41 api.go:302: aj6DvYcTrB8z3uPtw3xXZ7: New user attempt
[DEBUG] 00:16:43 api.go:361: Applying settings from profile "Default"
[DEBUG] 00:16:43 api.go:367: Applying policy from profile "Default"
[DEBUG] 00:16:43 api.go:375: Applying homescreen from profile "Default"
[GIN/DEBUG] 00:16:44: POST(/newUser) => 200 in 3.072307966s; 
```
The browser console log was empty, and all network requests returned `200`

**Configuration**
The only relevant configs settings I could think of: 
```ini
[jellyfin]
server        = https://jellyfin.<mydomain>.net
public_server = https://jellyfin.<mydomain>.net
```
(My Jellyfin server is publicly accessible at `https://jellyfin.<mydomain>.net`)

**Platform/Version**
Version 0.2.5
Commit fdc97b4
Running in Docker

Browser: tried in both Firefox 84.0.2 and Chromium 87.0.4280.88

**Comment by hrfee** [comment]
Completely overlooked the function of the button when I made it, it's not even a link 😂. I'll fix in the next version, should be out tomorrow.

**Comment by rigrig** [comment]
Great! (I was worried it would be some nasty bug of the "some buttons suddenly don't work for some users" kind ;-)

**Comment by hrfee** [comment]
fixed in 6860933, available in a bit on the `unstable` tag on dockerhub if you want it now.
